src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,24 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    public boolean isClosedWithinPastHour() {
```

Comment:
There are lot of repetitive code snippets in this method, indicating opportunities to abstract as utility method. Most of this code can be moved to a TimeHelper class. I remember we have a class like that, but forget the exact name. Most of this code belong in there as measuring the lapse between two time objects is a general function not specific to this class. 
Can do as a separate PR.